packages/react/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "@cerbos/react",
+  "version": "0.1.0",
```

Let's go with 0.1.0 for the initial release (I don't think we need to mark it as alpha/beta - we only did that for the embedded client because we didn't yet have full support for all possible policies in the WASM compiler).

However, with the new automation, we will do a minor version bump (as specified in the changelog) to the number we put here before actually releasing, so for now it should probably be:

```suggestion
  "version": "0.0.1",
```